moped-editor/netlify.toml

```diff
@@ -5,7 +5,7 @@
 #   From: https://docs.netlify.com/site-deploys/overview/#deploy-contexts
 #   Also: https://docs.netlify.com/configure-builds/environment-variables/
 [context.branch-deploy]
-    command = "echo 'Build Context Mode: Branch-Deploy' && npm run build:netlifypr"
+    command = "echo 'Build Context Mode: Branch-Deploy' && npm run build:netlifybranch"
```

Comment:
This is going to make this a bit of a leap of faith. Netlify deploys will not respect the new system until this is in the branches that are being deployed.